src/main/scala/forms/validation/DateTimeRangeValidator.scala

```diff
@@ -57,11 +58,12 @@ case class DateTimeRangeValidator(allowPast: Boolean, isEndDateMandatory: Boolea
       }
 
       val startDateV: ValidatedNel[FieldError, LocalDate] = dateValidator.validate(s"$path.startDate", sdv).leftMap {
-        v => NonEmptyList.of(FieldError(s"$path.startDate", mustProvideValidStartDateMessage))
+        v => v
```

Comment:
If you're not going to do anything to the value then there's no point in mapping it. But now your tests are failing